linux_os/guide/system/bootloader-grub2/grub2_admin_username/tests/grub-passwords.sh

```diff
@@ -0,0 +1,16 @@
+GRUB_CFG_ROOT=/boot/grub2
+
+function make_grub_password {
+	# password is "lala"
+	echo 'GRUB2_PASSWORD=grub.pbkdf2.sha512.10000.8F7F0A0D0F30D1924648F26D8A063A72373584C38DF22AEEC7B2C66329A47B04D16B92D73B58B36DB30E0AF85BD461F9AEB22BDE7290A9C212329BE21D2CDDEC.FDFD532D964C7F2EB2230A575C3630757502B2DC7A347987AABF64AE392EEDF3EEA86D6BCFAB8951F3003EA29435446964E54D2CC50AA0AC957C4B94BA8184FE' > "$GRUB_CFG_ROOT/user.cfg"
+}
+
+
+function set_superusers {
+	echo "set superusers=\"$1\"" > "$GRUB_CFG_ROOT/grub.cfg"
+}
+
+
+function remove_grub_password {
+	rm -f "$GRUB_CFG_ROOT/user.cfg"
+}
```

Comment:
Having a function that only renames rm command seems unnecessary. Moreover, its name is misleading.